src/styles/block/calendar.scss

```
@@ -0,0 +1,46 @@
+.calendar {
+  width: $calendar__width;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 10% auto;
+  padding: 0 $calendar__padding;
+  gap: $calendar__gap;
+
+  @for $day from 1 to $calendar__month-length {
+
+    &__day {
+      @include square($calendar__size);
+      box-sizing: border-box;
+      background-color: #eee;
+      border: 1px solid black;
+      text-align: center;
+      line-height: 100px;
+      cursor: pointer;
+      transition: 1s;
```

```suggestion
      transition: transform 1s;
```
better to specify a property in transition